docs/src/routes/docs/components/checkbox-chip-group.svx

```diff
@@ -0,0 +1,101 @@
+---
+name: CheckboxChipGroup
+relatedComponents:
+  - name: Chip
+    link: ./docs/components/chip
+  - name: RadioChip
+    link: ./docs/components/radio-chip
+  - name: RadioChipGroup
+    link: ./docs/components/radio-chip-group
+  - name: CheckboxChip
+    link: ./docs/components/checkbox-chip
+---
+
+<script>
+  import { CheckboxChipGroup } from 'attractions';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+  import RelatedComponents from 'src/components/docs/related-components.svelte';
+
+  const items = [
+    { value: 1, label: 'option 1', checked: true },
+    { value: 1.5, label: 'option 1.5', disabled: true },
+    { value: 2, label: 'option 2' },
+  ];
+</script>
+
+# CheckboxChipGroup
+
+A set of options presented as chips with multiple options to select.
+
+If you need to allow selecting only one option, consider using [==`RadioChipGroup`==](./docs/components/radio-chip-group). If you need more customization for the chips, such as their slots, take a look at [==`CheckboxChip`==](./docs/components/checkbox-chip), the building block of this component.
```

Comment:
> If you need more customization for the chips, such as their slots...

Maybe we can add a `<slot name="item">` to this component that passes it along to each chip

Reply:
Why? That's exactly the problem that the individual chips are meant to solve + the ability to lay them out in a manual way

Reply:
This is something in between. You want the features given by the Group component (the implicit `{#each}`, the styles, ...), but also want to customize the content of all chips at once.
I'm not saying it is for this PR, but it is worth considering for later

Reply:
There are barely any styles for the group, so individual components aren't losing much. The `{#each}`, pretty sure every Svelte developer knows how to write their own loop.
I don't like slots for group components like this one